src/test/java/org/fungover/haze/HazeDatabaseTest.java

```diff
@@ -159,5 +159,28 @@ void shouldShouldReturnTrue(){
         assertThat(testDatabase.containsKey("key1")).isTrue();
     }
 
+    @Test
```

Comment:
These tests should rather be placed in the HazeListTest class.
To setup testing we can use a method like hazeList.rPush(List.of("", "key1", "value1", "value2")); to create a key called key1 that is of type list and holds two values.